turingschool-examples/whats-cookin-starter-kit
# Feature/random recipes carousel

## Is this a fix or a feature?

Feature

## What is the change?

- Adds initial functionality to the homepage carousel - displays 5 randomly selected recipes as DOM cards.
- Adds initial functionality to the search bar - prints search results to console for now, rather than manipulating the DOM tree
- Other minor changes to improve search functionality, recipe calculations etc.

## What does it fix?

- Provides initial DOM manipulation and user interface.

## Where should the reviewer start?

- All code in scripts.js is new

## How should this be tested?

- Clone this repository and open index.html in browser. Refresh page to verify random recipe cards are populating the carousel each time.
- Enter sensible and non-sensible search terms into the search bar and review results on the console. using Chrome dev tools.

## Additional notes

- Need to incorporate logic to prevent duplicate recipe cards in the carousel.

## Relevant ScreenShots

![Screen Shot 2021-02-09 at 4 04 27 PM](https://user-images.githubusercontent.com/72054706/107452538-9cd34780-6b06-11eb-8ea2-674553bce85e.png)
